### PR TITLE
issue #665, removed back buttons

### PR DESCRIPTION
**Pull request by okwme**
`yarn run lint` and `yarn run test` both fail with exit code 1

### Issue 665

closes: ISSUE #665 

### Screenshots

not sure why the package.json formatting got changed. thought this might get reset when lint is run, but as noted above lint and test did not work. is that error a common sight / easy fix?

### CODE REVIEW

**Comment by nylira** [comment]
Yeah, the team uses [Prettier](https://github.com/prettier/prettier) for formatting our code. You should have it run in your editor after every save and it should solve it.

**Comment by NodeGuy** [comment]
BTW, `package.json` gets updated by Yarn when adding dependencies so Prettier is configured to ignore it.

File: package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
     "node": ">=9.4.0"
   },
   "scripts": {
+
```
Comment (jolesbi) [inline package.json]: can you remove these two new lines?

Comment (okwme) [reply package.json]: 👍

**Comment by jolesbi** [comment]
welcome to the team @okwme 🎉

**Pull request merged**